datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/SchemaTab.tsx

```diff
@@ -49,6 +55,60 @@ export const SchemaTab = ({ properties }: { properties?: any }) => {
     );
 
     const [showKeySchema, setShowKeySchema] = useState(false);
+    const [schemaViewMode, setSchemaViewMode] = useState(SchemaViewType.NORMAL);
+
+    const { data: getSchemaBlameVersionsData } = useGetSchemaBlameVersionsQuery({
+        skip: !datasetUrn,
+        variables: {
+            input: {
+                datasetUrn,
+                categories: [ChangeCategoryType.TechnicalSchema],
+            },
+        },
+    });
+    const latestVersion: string = getSchemaBlameVersionsData?.getSchemaBlame?.latestVersion?.semanticVersion || '';
+
+    const showSchemaBlame: boolean = schemaViewMode === SchemaViewType.BLAME;
+    const semanticVersionList: Array<SemanticVersionStruct> =
+        getSchemaBlameVersionsData?.getSchemaBlame?.semanticVersionList || [];
+    const version = useGetSemanticVersionFromUrlParams();
+    const selectedVersion = version || latestVersion;
+
+    const selectedSemanticVersionStruct = semanticVersionList.find(
+        (semanticVersion) => semanticVersion.semanticVersion === selectedVersion,
+    );
+    const selectedVersionStamp: string = selectedSemanticVersionStruct?.versionStamp || '';
+
+    let editMode = true;
+    if (selectedVersion !== latestVersion) {
+        editMode = false;
+    } else if (properties && properties.hasOwnProperty('editMode')) {
+        editMode = properties.editMode;
+    }
+
+    const { data: getSchemaBlameData } = useGetSchemaBlameQuery({
+        skip: !datasetUrn,
+        variables: {
+            input: {
+                datasetUrn,
+                version: selectedVersion,
+                categories: [ChangeCategoryType.TechnicalSchema],
```

Comment:
Categories not strictly required right? Just putting this for fun?

Reply:
Yes, this is just to make it more transparent what's going on and easier to change.

Reply:
Feels like "SchemaBlame" should be specific to technical schema changes. Unsure we should even support providing other categories. But will leave that up to you

Reply:
I thought programmatic users may decide they care about other things like doc changes. Will leave as is for now.